findMin.py

```diff
@@ -0,0 +1,73 @@
+
+#Time Complexity=O(log n)
+#Space Complexity=O(1)
+# Diffciulties faced: I am using min() here , does that affect the time complexity?
```

Comment:
Using built in methods do add up to complexity. 
In this case, using min is not advisable as you can just return min(nums) without even doing the binary search bit.